js/inputbar.js

```diff
@@ -359,8 +360,7 @@ weechat.directive('inputBar', function() {
 
                 // Alt-h -> Toggle all as read
                 if ($event.altKey && !$event.ctrlKey && code === 72) {
-                    var buffers = models.getBuffers();
-                    _.each(buffers, function(buffer) {
+                    _.each(models.getBuffers(), function(buffer) {
```

Comment:
It's kind of ridiculous to fix jshint by getting rid of this binding but honestly I find this solution preferable to renaming `buffers` above. Or maybe pull it up to the beginning of the function?